node/network/collator-protocol/src/validator_side/collation.rs

```diff
@@ -111,6 +114,19 @@ impl PendingCollation {
 	}
 }
 
+/// Result of advertisement seconding check.
+pub struct SecondingCheckResult {
```

Comment:
style: i find it a bit odd for this to be defined here when it is only used elsewhere